email-utils.js

```diff
@@ -18,65 +18,37 @@ const hbsOptions = {
   extName: ".hbs",
 };
 
-// This is set later when reading SMTP credentials from the environment.
-// This exists as a variable so we can use it in the from header of emails.
-let kSMTPUsername;
-
 // The SMTP transport object. This is initialized to a nodemailer transport
 // object while reading SMTP credentials, or to a dummy function in debug mode.
 let gTransporter;
 
 const EmailUtils = {
-  init() {
-    // Allow a debug mode that will send JSON back to the client instead of sending emails.
-    // eslint-disable-next-line no-process-env
-    if (process.env.DEBUG_DUMMY_SMTP) {
-      console.info("Running in dummy SMTP mode, /user/breached will send a JSON response instead of sending emails.");
+  async init(smtpUrl = AppConstants.SMTP_URL) {
+
+    // Allow a debug mode that will log JSON instead of sending emails.
+    if (smtpUrl === "") {
```

Comment:
ANy benefits to checking an empty string vs just falsy value, a la `if (!smtpUrl) {..}`?